wagtail/admin/views/pages/bulk_actions/move.py

```diff
@@ -43,7 +43,7 @@ def __init__(self, request, model):
 
     def get_form_kwargs(self):
         ctx = super().get_form_kwargs()
-        ctx["destination"] = self.destination or Page.get_first_root_node()
+        ctx["destination"] = self.destination
```

Comment:
I don't recall the exact reason we went for this, but it was something to do with how the non-bulk chooser worked at the time which we tried to follow as close as possible